web_invalid_tab/static/src/js/view_form.js

```diff
@@ -0,0 +1,62 @@
+openerp.web_invalid_tab = function(instance) {
+
+    var tab_selector = 'div[role="tabpanel"]';
+
+    function tab_link(tab) {
+        return $("a[href='#" + tab.attr('id') + "']");
+    }
+
+    function is_visible(tab, element) {
+        if ($(element).hasClass('oe_form_invisible') || element.style.display == 'none') {
```

Comment:
@hparfr could you please check the last comment? I would like to know if I can improve the code in some way. Thanks